config/default.yaml

```diff
@@ -0,0 +1,48 @@
+---
+
+auth:
+    # A public key for verifying JWTs signed by api.screwdriver.cd
+    jwtPublicKey: |
+        -----BEGIN PUBLIC KEY-----
+        INSERT STUFF HERE
+        -----END PUBLIC KEY-----
+
+httpd:
+    # Port to listen on
+    port: 80
+    # Host to listen on (set to localhost to only accept connections from this machine)
+    host: 0.0.0.0
+    # Externally routable URI (usually your load balancer or CNAME)
+    uri: http://localhost
+    # SSL Support
+    tls: false
+        # If you want SSL, you can easily add it by replacing `tls: false` with an object that
+        # provides the options required by `tls.createServer`
+        # https://nodejs.org/api/tls.html#tls_tls_createserver_options_secureconnectionlistener
+        # key: |
+        #     PRIVATE KEY HERE
+        # cert: |
+        #     YOUR CERT HERE
+
+builds:
+    # How long should builds stay around for
```

Comment:
The `maxByteSize` implies that it's in the units of bytes. `expiresIn` doesn't explicitly specify seconds or milliseconds.